src/components/PrivateRoute.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import { Route, Redirect } from "react-router-dom";
 
+import routes from '../constants/routes';
+
 const PrivateRoute = ({ component: Component, ...rest }) => (
   <Route
     {...rest}
     render={props =>
       true ? (
         <Component {...props} />
       ) : (
-        <Redirect to={{ pathname: '/login' }} />
+        <Redirect to={{ pathname: routes.home }} />
```

Comment:
shouldn't this be routes.login? I mean rn it's the same since it never reaches this part of the condition but you are changing the route.

Reply:
You are right 👍